src/EnergyPlus/PoweredInductionUnits.cc

```diff
@@ -1608,6 +1593,9 @@ namespace PoweredInductionUnits {
 		QZnReq = ZoneSysEnergyDemand( ZoneNum ).RemainingOutputRequired;
 		QToHeatSetPt = ZoneSysEnergyDemand( ZoneNum ).RemainingOutputReqToHeatSP;
 		CpAirZn = PsyCpAirFnWTdb( Node( ZoneNode ).HumRat, Node( ZoneNode ).Temp );
+		// Initialize local fan flags to global system flags
+		PIUTurnFansOn = ( DataHVACGlobals::TurnFansOn || DataHVACGlobals::TurnZoneFansOnlyOn );
+		PIUTurnFansOff = DataHVACGlobals::TurnFansOff;
```

Comment:
Doesn't seem like this local is currently really being used anywhere here.  But I don't think it causes an issue, and it does set a nice pattern along with the local `On` variable.